### ISSUE TITLE
Add an option to change aria-live setting.

### ISSUE THREAD

**Issue by dgrumbler** [ganlanyuan/tiny-slider#659]
I noticed when testing this slider on safari voiceover that the carousel keeps interrupting to tell the user it has changed slider. While very useful this can also be quite annoying for a non sighted user who is trying to listen through the page if used with the autoplay feature for example. It would be good if we could choose to change the aria-live setting for situations where the slider is rapid and could be annoying to the user.

Quote from : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
"Normally, only aria-live="polite" is used. Any region which receives updates that are important for the user to receive, **but not so rapid as to be annoying**, should receive this attribute. The screen reader will speak changes whenever the user is idle."

**Comment by basher** [comment]
Is this going to be fixed?

According to https://www.w3.org/WAI/ARIA/apg/patterns/carousel/examples/carousel-1-prev-next/, `aria-live="off"` should be set for auto-rotating carousels, and only set to `"polite"` when user interacts.
